scripts/bootstrap

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env bash
+
+#
+# bootstrap
+#
+# Install and configure Opkg for use with Entware and Toltec
+#
+
+bootstrap_wget() {
+    if [ ! -d /home/root/.cache/wget_bin ]
+    then
+        log INFO "Installing a current wget binary with better and current tls support" \
+            "to /home/root/.cache/wget_bin/ for this installation..." \
+            ""
+        # Bootstrap a current version of wget
+        WGET_BINARIES_PATH='http://github.com/LinusCDE/wget-remarkable-pipeline/releases/download/job254'
+        WGET_BINARIES_FILENAME='wget-remarkable-pipeline_job245_wget1.20.3.zip'
+        WGET_BINARIES_SHA256='84185a5934e34e25794d439c78dc9f1590e4df12fbf369236f6a8749bf14d67f'
+
+        # Download and compare to hash
+        wget "$WGET_BINARIES_PATH/$WGET_BINARIES_FILENAME" -O "/home/root/$WGET_BINARIES_FILENAME"
+        if ! echo "$WGET_BINARIES_SHA256  /home/root/$WGET_BINARIES_FILENAME" | sha256sum -c -
+        then
+            echo "FATAL: Invalid hash" >&2
+            exit 1
+        fi
+
+        # Ensure to /home/root/.cache/wget_bin exists and is empty
+        if [ -d /home/root/.cache/wget_bin ]
+        then
+            rm -rf /home/root/.cache/wget_bin/*
```

Comment:
This seems dangerous: what if the user happens to have existing data in a folder named like this? I think the script should fail if the directory already exists.

Reply:
It seems pretty unlikely that a user would happen to have such a directory to me. Though you're right that this is a concern. Since I added a check for that path to be a file. Doing a rm on a directory is unnecessary and a error for a file is a good idea.

Addressed in commit d8731cd.